src/components/dataDisplays/cellRenderers/OverflowController.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { debounce } from 'lodash-es';
+
+import Tooltip from '@material-ui/core/Tooltip';
+
+const noWrapStyles = {
+  display: 'block',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+};
+
+export default function OverflowController({ title, children }) {
+  const [isOverflowed, setIsOverflowed] = useState(false);
+  const [refEl, setRefEl] = useState();
+
+  const ref = useCallback(node => {
```

Comment:
1. From the React docs: ([useCallback](https://reactjs.org/docs/hooks-reference.html#usecallback)):
> `useCallback(fn, deps)` is equivalent to `useMemo(() => fn, deps)`.
2. It shouldn't be called continuously because of the way that refs are handled.
> React will call the ref callback with the DOM element when the component mounts, and call it with null when it unmounts. ([callback refs](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs))